src/core/binding.c

```diff
@@ -1034,6 +1034,7 @@ QuicBindingProcessStatelessOperation(
         SendDatagram->Length =
             QuicPacketEncodeRetryV1(
                 RecvPacket->LH->Version,
+                TRUE, // TODO : Find a way to check connection settings and PeerTPs.
```

Comment:
I think when writing all stateless packets (Retry, Reset) we're going to always set the fixed bit (not grease it) because we're never going to have any connection-specific state to base this decision on. So, really, I think we might as well, undo the changes to these encode function and just have them stay fixed.

Reply:
Alright! I'm going to undo it to the original signature.